paddle/cinn/backends/compiler.h

```diff
@@ -32,6 +33,34 @@
 namespace cinn {
 namespace backends {
 
+/**
+ * A class for dumping the code after compilation.
+ * Use FLAGS_cinn_dump_group_lowered_func to specify the directory to dump
+ * lowered function. Use FLAGS_cinn_dump_group_source_code to specify the
+ * directory to dump the source code. Use FLAGS_cinn_dump_group_ptx to specify
+ * the directory to dump ptx. Use FLAGS_cinn_dump_group_instruction to specify
+ * the directory to dump instruction.
+ */
+class DumpCompilationInfo {
```

Comment:
Although there are exceptions, but modern software engineer encourages class to be "noun" while method to be "verb" (so we call `object` as the instance of class, not `action`)

For example:
`Executor.run`
`Logger.log`

So I suggest you change class name to `CompilationInfoDumper`